src/contexts/W3iContext/hooks/dappOrigin.ts

```diff
@@ -12,5 +12,20 @@ export const useDappOrigin = () => {
   const [dappIcon] = useState<string>(dappIconQuery ?? '')
   const [dappNotificationDescription] = useState<string>(dappNotificationDescriptionQuery ?? '')
 
+  if (dappOrigin) {
+    const cssId = 'web3inbox-external-style'
+    const doc = document
+    const head = doc.getElementsByTagName('head')[0]
+    const link = doc.createElement('link')
+    link.id = cssId
+    link.rel = 'stylesheet'
+    link.type = 'text/css'
+
+    link.href = `${dappOrigin}/web3inbox.css`
+    link.media = 'all'
+    doc.documentElement.style.setProperty('--accent-color-1', 'red')
```

Comment:
Smells a bit like a debug prop? Can't tell at face value 😄

Reply:
Yes, yes it is. Removed